src/containers/Tenant/Diagnostics/Overview/TopicStats/TopicStats.scss

```diff
@@ -0,0 +1,33 @@
+.ydb-overview-topic-stats {
+    .ydb-loader {
+        margin-top: 50px;
+    }
+
+    &__info {
+        .info-viewer {
```

Comment:
if you can define block styles without referencing other blocks, better do this

works:
```
<div className="block1">
    <div className="block1__elem">
        <div className="block2" />
    </div>
</div>

.block1__elem .block2 {
}
```

works better:
```
<div className="block1">
    <div className="block1__elem block2" />
</div>

.block1__elem {
}
```

or even if you need nesting for some reason
```
<div className="block1">
    <div className="block1__elem1">
        <div className="block1__elem2 block2" />
    </div>
</div>

.block1__elem2 {
}
```

Reply:
an example to compare assumptions
```
.block {
    // I have an element inside me, it is implemented using block ydb-text, its color id red
    .ydb-text {
        color: red;
    }
}

.block {
    // I have an element A, its color is red
    &__text {
        color: red;
    }
}
```